articles/vulnerability-processing.md

```diff
@@ -21,7 +21,7 @@ Fleet detects vulnerabilities for these software types:
 | Packages            | Python, Homebrew  | Python, Atom, Chocolatey | <p>For Ubuntu, Debian, RHEL (including CentOS), and Fedora: packages defined in the [OVAL definitions](https://github.com/fleetdm/nvd/blob/master/oval_sources.json), except for vulnerabilities involving configuration files.</p><p>For Amazon Linux, packages maintained by Amazon by checking [ALAS advisories](https://alas.aws.amazon.com/).</p> |
 | IDE extensions      | VS Code extensions | VS Code extensions | VS Code extensions                                                                                                                                                                                                                                                                                                                                     |
 
-As of right now, only app names with all ASCII characters are supported. Apps with names featuring non-ASCII characters, such as Cyrillic, will not generate matches.
+Currently, only app names with all ASCII characters are supported. Apps with names featuring non-ASCII characters, such as Cyrillic, will not generate matches. Software that has been renamed from its default name (e.g. "Chrome 2" instead of "Google Chrome") may not generate matches, though Fleet includes [custom matching rules](https://github.com/fleetdm/fleet/blob/main/server/vulnerabilities/nvd/cpe_translations.json) to mitigate common known issues on an app-by-app basis.
```

Comment:
```suggestion
Currently, only software names with all ASCII characters are supported. Vulnerabilities won't be detected for software with names featuring non-ASCII characters, such as Cyrillic, nor software that has been renamed from its default name (e.g. "Chrome 2" instead of "Google Chrome"). For some software, Fleet uses [custom rules](https://github.com/fleetdm/fleet/blob/main/server/vulnerabilities/nvd/cpe_translations.json) to mitigate these issues on an app-by-app basis.
```

Language tweak to make this about "detecting vulnerabilities" rather than "matching":

![Screenshot 2025-01-30 at 9 03 18 AM](https://github.com/user-attachments/assets/41e0ecae-fab0-4596-943c-eca12fd7a23d)